tests/unittests/RandomTest.cpp

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "glow/Support/Random.h"
+
+#include "gtest/gtest.h"
+
+using namespace glow;
+
+// Test that nextRandInt generates every number in the closed interval [lb, ub].
+// Use enough trials that the probability of random failure is < 1.0e-9.
+TEST(Random, closedInterval) {
+  constexpr int lb = -3;
+  constexpr int ub = 3;
+  constexpr int trials = 200;
+
+  for (int i = lb; i < ub; i++) {
```

Comment:
i <= ub ? nextRandInt should generate in [lb, ub]

Reply:
Wow.  That is super embarrassing :).  Thanks for the catch.